tests/DCPS/InconsistentTopic/InconsistentTopic.mpc

```diff
@@ -22,4 +14,13 @@ project(DDS*PubblisherSubscriber) : dcpsexe, dcps_transports_for_test {
   Source_Files {
     pubsub.cpp
   }
+
+  idlflags      += -Wb,stub_export_include=InconsistentTopic_export.h \
```

Comment:
Nothing related to "export" is needed here since there's no library being built.